nodeletctl/pkg/nodeletctl/deployer.go

```diff
@@ -397,6 +411,30 @@ func (nd *NodeletDeployer) DeleteNodelet() error {
 	return nil
 }
 
+func (nd *NodeletDeployer) DeleteOldCerts() error {
+	deleteCmd := "rm -rf /etc/pf9/kube.d/certs"
+	if _, _, err := nd.client.RunCommand(deleteCmd); err != nil {
+		return fmt.Errorf("Failed to remove old certs: %s", err)
+	}
+	return nil
+}
+
+func (nd *NodeletDeployer) RegenCerts(wg *sync.WaitGroup) error {
```

Comment:
Rename to `UploadCertsAndRestartStack`. `RegenCerts` is misleading in that it does not indicate that entire stack will be restarted